Code/Editor/Plugins/EditorAssetImporter/AssetImporterWindow.cpp

```diff
@@ -260,12 +275,19 @@ SceneSettingsCard* AssetImporterWindow::CreateSceneSettingsCard(
     SceneSettingsCard::Layout layout,
     SceneSettingsCard::State state)
 {
+    QLayoutItem* previousCard;
+    while ((previousCard = ui->m_cardAreaLayout->takeAt(0)) != nullptr)
+    {
+        delete previousCard->widget();
+        delete previousCard;
+    }
+
```

Comment:
Consider renaming this something like currentCard or cardToDelete. If possible move it into the while() unless it's used elsewhere.